wled00/bus_manager.h

```diff
@@ -315,6 +324,37 @@ class BusNetwork : public Bus {
     bool      _broadcastLock;
 };
 
+#ifdef WLED_ENABLE_HUB75MATRIX
+class BusHub75Matrix : public Bus {
+  public:
+    BusHub75Matrix(BusConfig &bc);
+    void setPixelColor(unsigned pix, uint32_t c) override;
```

Comment:
This is the place for hot attribute as the compiler will know to optimise everywhere.

Reply:
Added